spleeter_service/spleeter_service.py

```
@@ -0,0 +1,76 @@
+            output_dir = 'output'
+            os.makedirs(output_dir, exist_ok=True)
+
+            response = {}
+            try:
+                # Use Spleeter
+                spleeter_command = ['spleeter', 'separate', '-i', filename, '-o', output_dir]
+                subprocess.run(spleeter_command, check=True)
+
+                # Use Demucs for guitar only
+                demucs_command = ['demucs', filename, '--two-stems', 'guitar']
+                subprocess.run(demucs_command, check=True)
```

Ruff (0.8.0)

`subprocess` call: check for execution of untrusted input

(S603)